src/scss/foundations/breakpoints.scss

```
@@ -28,101 +28,180 @@ $_breakpoints: (
   xlarge: $_breakpointXlarge,
 );
 
+/// The breakpoints of Pyxis.
+///
+/// @type Map
+/// @prop {Number} base [1px]
+/// @prop {Number} xxsmall [375px]
+/// @prop {Number} xsmall [576px]
+/// @prop {Number} small [768px]
+/// @prop {Number} medium [992px]
+/// @prop {Number} large [1200px]
+/// @prop {Number} xlarge [1680px]
+///
+/// @access public
+$breakpoints: ();
```

Su sassodc è molto comodo avere questa mappa con le prop esposte, ma eviterei di avere una variabile pubblica, potremmo modificare in qualcosa tipo:

```suggestion
/// The breakpoints of Pyxis.
/// @hidden This variable is useful only in sassdoc documentation
///
/// @type Map
/// @prop {Number} base [1px]
/// @prop {Number} xxsmall [375px]
/// @prop {Number} xsmall [576px]
/// @prop {Number} small [768px]
/// @prop {Number} medium [992px]
/// @prop {Number} large [1200px]
/// @prop {Number} xlarge [1680px]
$_breakpointProps: ();
```

In questo modo non esponiamo la variabile, ma sassdoc continua comunque a renderizzarla correttamente (dato che non specifichiamo `@access`). Non è una soluzione meravigliosa, ma al momento non mi viene in mente nulla di meglio 🤔 

Farei questa modifica per tutte le variabili vuote con lo stesso pattern nei vari file (`$colors`, `$spacings`, ecc...)